src/ProjectTemplates/Web.ProjectTemplates/content/EmptyComponentsWebAssembly-CSharp/.template.config/ide.host.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json.schemastore.org/vs-2017.3.host",
+  "order": 610,
```

Comment:
We need to bump up the order probably if it hasn't already been changed from the default template. `660` given empty server was `650`.